New version: HTMLTables v0.4.4

JuliaRegistrator opened the pull request:
- Registering package: HTMLTables
- Repository: https://github.com/cecoeco/HTMLTables.jl
- Created by: @cecoeco
- Version: v0.4.4
- Commit: bb7ec4fa83ea28b40788a21981498849eaf6f0b2
- Git reference: HEAD
- Description: Julia package for reading and writing HTML tables using the Tables.jl interface

cecoeco commented:
I added the wrong version number and wish to close this request, since I have another pull request with the right version.

JuliaHubRegistrator committed:
New version: HTMLTables v0.4.4

UUID: b1afcece-b80e-4563-b90e-36b4cc56d3fa
Repo: https://github.com/cecoeco/HTMLTables.jl.git
Tree: 0845af83c0198eb7a882456ea15e7e7becca8bf9

Registrator tree SHA: 191228b6dd8b9d0e2965ae3e705fe54c51dcfee8
